tuf/store/memorystore.go

```diff
@@ -95,16 +119,16 @@ func (m *memoryStore) WalkStagedTargets(paths []string, targetsFn targetsWalkFun
 	return nil
 }
 
-func (m *memoryStore) Commit(map[string][]byte, bool, map[string]data.Hashes) error {
+func (m *MemoryStore) Commit(map[string][]byte, bool, map[string]data.Hashes) error {
 	return nil
 }
 
-func (m *memoryStore) GetKey(role string) ([]byte, error) {
-	return nil, fmt.Errorf("GetKey is not implemented for the memoryStore")
+func (m *MemoryStore) GetKey(role string) ([]byte, error) {
+	return nil, fmt.Errorf("GetKey is not implemented for the MemoryStore")
 }
 
 // Clear this existing memory store by setting this store as new empty one
-func (m *memoryStore) RemoveAll() error {
+func (m *MemoryStore) RemoveAll() error {
 	m.meta = make(map[string][]byte)
 	m.files = make(map[string][]byte)
 	m.keys = make(map[string][]data.PrivateKey)
```

Comment:
Also, it could be nice to have a small unit test for the `MemoryStore` focusing on `GetMeta`/`SetMeta` when we include hashes and potentially the `RemoveAll` and `RemoveMeta` functionality if those change.